win32ss/user/ntuser/defwnd.c

```diff
@@ -945,6 +945,24 @@ IntDefWindowProc(
                    wParamTmp = UserGetKeyState(VK_SHIFT) & 0x8000 ? SC_PREVWINDOW : SC_NEXTWINDOW;
                    co_IntSendMessage( Active, WM_SYSCOMMAND, wParamTmp, wParam );
                 }
+                else if (wParam == VK_SHIFT) // Alt+Shift
+                {
+                    RTL_ATOM ClassAtom = 0;
+                    UNICODE_STRING ustrClass, ustrWindow;
+                    HWND hwndSwitch;
+
+                    RtlInitUnicodeString(&ustrClass, L"kbswitcher");
```

Comment:
Can you prove this is compatible with Windows?

Reply:
> Can you prove this is compatible with Windows?

Nope. Windows doesn't have `kbswitch` program. Windows XP/2k3 has `ctfmon.exe` instead.

Reply:
True, but my question was more about hardcoding a window class name in win32k to implement what you're doing, instead of e.g. using a flag being set/unset, or a keyboard hook, etc.

Reply:
No. No such a window class in Windows.

Reply:
You want me to implement CTF server?

Reply:
> You want me to implement CTF server?

:sweat_smile: that's far beyond the purpose of fixing the alt-shift ^^ Our kbswitch.exe seems to be more kind of closely related to what windows 2000 may have, as far as I remember.